### PR TITLE
Setup logging

### PR DESCRIPTION
**Pull request by mpetrun5**
- add logger with levels
- added sending logs to file option

### CODE REVIEW

File: internal/lb/ping.go [hunk 1]
```diff
@@ -30,12 +31,12 @@ func (ps *pingService) Send() (*http.Response, error) {
 		Timestamp: time.Now().Unix(),
 	}
 
-	log.Println("Sending ping to load balancer")
+	log.Info("Sending ping to load balancer")
```
Comment (mpetrunic) [inline internal/lb/ping.go:34]: verbose?

File: internal/lb/metrics.go [hunk 2]
```diff
@@ -27,12 +28,12 @@ func (ms *metricsService) Send(client node.Client) (*http.Response, error) {
 		return nil, err
 	}
 
-	log.Println("Sending metrics to load balancer")
+	log.Info("Sending metrics to load balancer")
```
Comment (mpetrunic) [inline internal/lb/metrics.go:31]: you should probably log some important metrics

File: internal/lb/ping.go (hunk 1 above)
Comment (mpetrun5) [reply internal/lb/ping.go:33]: Added log levels so shoule be fine

**Review by mpetrunic — APPROVED** [review]

File: internal/lb/ping.go (hunk 1 above)
Comment (mpetrunic) [reply internal/lb/ping.go:33]: no wait it's not ok, this shouldn't be written on info level every 5 seconds lol.

Should be log.Verbose

**Review by mpetrunic — CHANGES_REQUESTED** [review]

File: internal/run/run.go [hunk 3]
```diff
@@ -16,7 +17,7 @@ func Start(lbClient *lb.Client, nodeClient node.Client, telemetry telemetry.Tele
 	if err != nil {
 		return err
 	}
-	log.Printf("Registered to load balancer %s", lbClient.BaseURL.String())
+	log.Info(fmt.Sprintf("Registered to load balancer %s", lbClient.BaseURL.String()))
```
Comment (MakMuftic) [inline internal/run/run.go:21]: There should be `log.Infof` method, so you can omit this `fmt.Sprintf` call.

**Review by MakMuftic — CHANGES_REQUESTED** [review]

**Review by MakMuftic — APPROVED** [review]

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**